pool-payout-zkapp/src/PoolPayout.ts

```diff
@@ -81,22 +92,41 @@ export class PoolPayout extends SmartContract {
       setZkappUri: Permissions.impossible(),
     });
 
-    // Move this to an init() method but this is useful to redeploy
     this.currentEpoch.set(Field(39));
     this.currentIndex.set(Field(0));
     this.feePercentage.set(UInt32.from(5));
     this.oraclePublicKey.set(PublicKey.fromBase58(ORACLE_PUBLIC_KEY));
     this.validatorPublicKey.set(PublicKey.fromBase58(VALIDATOR_PUBLIC_KEY));
   }
 
+  @method
+  updateEpoch(n: Field) {
```

Comment:
Adding these methods for tests.  Probably bad to push to production like this, potentially worth considering though... depends how much power you want to give the contract owner.